src/ResourceManager/Automation/Commands.Automation.Test/ScenarioTests/AutomationTestCommon.ps1

```diff
@@ -0,0 +1,147 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Create Automation account and return the account
+#>
+function CreateAutomationAccount
+{
+	$resourceGroupName = Get-RandomResourceGroupName
+	$location = Get-AutomationAccountTestLocation
+	$automationAccountName = Get-RandomAutomationAccountName
```

Comment:
Please clean up the formatting - use spaces instead of tabs.

Reply:
Resolved